kie-api/src/main/java/org/kie/api/KieServices.java

```diff
@@ -316,9 +316,16 @@
     Environment newEnvironment();
 
     /**
+     * Returns a reference to the KieServices singleton
+     */
+    static KieServices get() {
+        return Factory.get();
+    }
+
+    /**
      * A Factory for this KieServices
      */
-    public static class Factory {
+    class Factory {
```

Comment:
No, breaking change: public static are just redundant in an interface. The class is public and static by default.